atlp-rwanda/script-heroes-frontend
# #174797459 Setup Sass

#### What does this PR do?
Configure the project to use Sass
#### Description of Task to be completed?
- Install `node-sass` and `sass-loader` as project dependencies
- Define webpack rules to allow sass files to work with the project
- Create a `.scss `file
#### How should this be manually tested?
- Run `yarn install` to get all project dependencies
- Write some styles in `App.scss`
- Go to the browser to observe changes
#### What are the relevant pivotal tracker stories?
[#174797459](https://www.pivotaltracker.com/n/projects/2465784/stories/174797459)
